src/public/js/constants.js

```diff
@@ -6,4 +6,99 @@ export const roomModes = {
   egalitarian: EGALITARIAN_MODE,
   directAction: DIRECT_ACTION_MODE,
   facilitated: FACILITATED_MODE,
-};
\ No newline at end of file
+};
+
+export const systemConfirmMsg_archivalRoom = {
```

Comment:
this is awesome! seems like a great way to add all the slides and keep them all in the same place